optuna_dashboard/ts/components/Artifact/StudyArtifactCards.tsx

```diff
@@ -24,6 +24,7 @@ import { StudyDetail } from "ts/types/optuna"
 import { actionCreator } from "../../action"
 import { ArtifactCardMedia } from "./ArtifactCardMedia"
```

Comment:
When a csv table is large, it is too long to display all 50 rows.
<img width="850" alt="スクリーンショット 2024-05-29 15 13 31" src="https://github.com/optuna/optuna-dashboard/assets/83964523/2a17fbbb-a904-4f10-b1dd-514b36e37c9c">
It would be better if you fix max height or something.

Reply:
Stop showing table here. Just leave file open button.